File: articles/decision-under-uncertainty-and-computational-risk/go/decision_risk_example.go
package main

import "fmt"

func expectedNetValue(p, benefit, loss, cost float64) float64 {
    if p < 0 { p = 0 }
    if p > 1 { p = 1 }
    return p*benefit - p*loss - cost
}

func main() {
    fmt.Printf("%.6f
", expectedNetValue(0.42, 150.0, 80.0, 25.0))
}

File: articles/decision-under-uncertainty-and-computational-risk/julia/decision_risk_example.jl
# Compact Julia example: expected net value.
function expected_net_value(p, benefit, loss, cost)
    p = clamp(p, 0.0, 1.0)
    return p * benefit - p * loss - cost
end

println(expected_net_value(0.42, 150.0, 80.0, 25.0))
